src/components/ReactTags.js

```diff
@@ -396,7 +406,11 @@ class ReactTags extends Component {
       placeholder = this.props.placeholder,
       inputName = this.props.name,
       inputId = this.props.id,
-      maxLength = this.props.maxLength;
+      maxLength = this.props.maxLength,
+      inline = this.props.inline,
+      inputFieldPosition = this.props.inputFieldPosition;
```

Comment:
replace lines 409-411 with below
```
const { maxLength, inline, inputFieldPosition } = this.props;
```

Reply:
Done. We can also rewrite this block as 
const { 
        placeholder,
        name: inputName,
        id: inputId,
        maxLength, 
        inline, 
        inputFieldPosition 
      } = this.props;
Let me know your thoughts.

Reply:
@atiqueansari1987 yes please go for it 👍